client/packages.go

```diff
@@ -61,10 +62,25 @@ type ResultInfo struct {
 	SuggestedCurrency string `json:"suggested-currency"`
 }
 
+type ListOptions struct {
+	SelectRefresh bool
```

Comment:
Sorry for going back and forth here.. can we please just say `Refresh: true` here? If it's good for the command line, it should be good for the Go API.. if it's bad, we should fix both as the CLI is much more important as a user-visible interface.